config/prod.exs

```diff
@@ -55,3 +55,7 @@ config :logger, level: :info
 # Finally import the config/prod.secret.exs which loads secrets
 # and configuration from environment variables.
 import_config "prod.secret.exs"
+
+config :google_crawler, GoogleCrawler.Tokenizer,
+  issuer: "google_crawler",
+  secret_key: System.get_env("GUARDIAN_SECRET")
```

Comment:
how about moving this to `prod.secret.exs` @hanam1ni

Reply:
Good idea 👍  In Omiselab, we don't use much of that file.

Reply:
Updated in this commit 710e3cba129dba0ff7c60bc9878040734bde869f